### ISSUE TITLE
2.3: Missleading error message about failed download of library

### ISSUE THREAD

**Issue by ggee** [DCC-EX/exInstaller#11]
When running the 'Compile and Upload', some messages are printed about 'Failed to download library name'.

```
Downloading arduino:avr@1.8.3: 4000974/4941548
Downloading arduino:avr@1.8.3: 4602102/4941548
Downloading arduino:avr@1.8.3: 4941548/4941548
arduino:avr@1.8.3 download complete
Failed to download library name
Obtaining https://github.com/Locoduino/DIO2.git via git
CommandStation EX Code obtained from Repository
Failed to download library name
Obtaining https://github.com/sparkfun/SparkFun_External_EEPROM_Arduino_Library.git via git
CommandStation EX Code obtained from Repository

```

1. 'library name' should be replaced by the actual name

2.  If it is related to DIO2 and EERPOM, this error seems to be wrong since the libraries did get downloaded into a libraries folder.

3. Or if it is complaining about other libraries not downloaded, what were they?  Are DIO2 and EEPROM the only git downloads?

**Comment by ggee** [comment]
The errors are more clear in 2.3.1 now.  At least it tells me it failed. Don't know why, but at least I know more.

```
Attempting to download avr:arduino
Obtaining https://github.com/Locoduino/DIO2.git via git
Failed to download library by name due to Status(StatusCode="Unknown", Detail="looking for library: library SparkFun External EEPROM Arduino Library not found") attempting to obtain via git
Obtaining https://github.com/sparkfun/SparkFun_External_EEPROM_Arduino_Library.git via git
Failed to download library by name due to Status(StatusCode="Unknown", Detail="looking for library: library SPI not found") attempting to obtain via git
Obtaining https://github.com/arduino-libraries/Ethernet via git
Obtaining https://github.com/arduino-libraries/Ethernet.git via git
Obtaining https://github.com/mathertel/LiquidCrystal_PCF8574.git via git
Changing MotorShield options
```

**Comment by ggee** [comment]
Original problem fixed in #13
